primitives/avn-common/src/lib.rs

```diff
@@ -38,6 +38,10 @@ pub const ETHEREUM_PREFIX: &'static [u8] = b"\x19Ethereum Signed Message:\n32";
 pub const EXTERNAL_SERVICE_PORT_NUMBER_KEY: &'static [u8; 15] = b"avn_port_number";
 /// Default port number the external service runs on.
 pub const DEFAULT_EXTERNAL_SERVICE_PORT_NUMBER: &str = "2020";
+/// Bound used for Vectors containing validators
+pub type MaximumValidatorsBound = ConstU32<256>;
```

Comment:
If you add these types on the config of the pallets, you can query them from polkadotJs and via a script. The way these are set, we would have to see the code to know what the limit is which makes it harder for DApps to validate data.

Reply:
true, this will be the case in some of the pallets where there the bounded data input or output comes from a user.
So far, the operations refactored are either sudo or in the case of the voting, an internal mechanism that dapps and users don't use. It is a better fit to use const values in the code I think

Reply:
My suggestion was not related to the source of the bounded data but rather to making that accessible in a running chain. For example if you want to know the max number of validators the voting logic supports, you would have to find the code and check the value. My suggestion is to make that value accessible in the chain state so you can read it from polkadotJS or from a script programatically. This approach is DApp friendly, user friendly and incurs no extra cost. 
![image](https://github.com/Aventus-Network-Services/avn-parachain/assets/39748285/c70b066b-868b-447b-814a-47c5bf31e2bc)

Reply:
Discussed offline and agreed to make this change in a separate PR